docs/reinstallation/image_based_break_fix.md

```diff
@@ -0,0 +1,99 @@
+# Image-Based Break/Fix (IBBF) for SNO Hardware Replacement
```

Comment:
Why have a whole separate doc for this?

Reply:
The flow for IBBF is slightly different with different checks - hence a separate, dedicated document to address it. Furthermore, it is also the primary requested feature from which the cluster reinstallation feature was built.

Reply:
@carbonin we decided to incorporate the IBBF doc under the same README.md. This avoids duplication of content.